src/components/PreloadFeatureFlags.tsx

```diff
@@ -0,0 +1,23 @@
+import { unstable_evaluate as evaluate } from "@vercel/flags/next"
```

Comment:
Cool realization! I’m just concerned about Next.js marking this as an unstable feature. What pitfalls could arise if they decide to change it?

Reply:
Very unlikely they change it drastically. Nowadays it's very common `unstable_` even in React itself.